packages/components/src/Modal/docs/controlled.tsx

```diff
@@ -1,18 +1,18 @@
-import { Button, Content, Heading, Modal, ModalTrigger } from "@hopper-ui/components";
+import { Button, Content, Heading, Modal } from "@hopper-ui/components";
 import { useState } from "react";
 
 export default function Example() {
     const [isOpen, setIsOpen] = useState(false);
 
     return (
-        <ModalTrigger isOpen={isOpen} onOpenChange={setIsOpen}>
-            <Button variant="secondary">Open modal</Button>
-            <Modal>
+        <>
```

Comment:
This is not the right way, the way it was before is better, you need the modal trigger?